hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/key/OMDirectoriesPurgeResponseWithFSO.java

```diff
@@ -130,11 +130,10 @@ public void processPaths(OMMetadataManager omMetadataManager,
         }
 
         RepeatedOmKeyInfo repeatedOmKeyInfo = OmUtils.prepareKeyForDelete(
-            keyInfo, null, keyInfo.getUpdateID(), isRatisEnabled);
+            keyInfo, keyInfo.getUpdateID(), isRatisEnabled);
 
         String deletedKey = omMetadataManager
-            .getOzoneKey(keyInfo.getVolumeName(), keyInfo.getBucketName(),
```

Comment:
For FSO, here we are using `/volumeId/bucketId/parentObjId/fileName/objectId` but`OMKeyDeleteResponseWithFSO` still uses `/volumeName/bucketName/keyName/objectId`
https://github.com/apache/ozone/blob/5eead92666b374c4bb912f332488acb23f9a81df/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/key/OMKeyDeleteResponseWithFSO.java#L92-L102